File: pset1/cash.c
#include<cs50.h>
#include<stdio.h>
#include<math.h>

int main(void)
{
    float change;
    do
    {
        change=100*get_float("Change owed: ");  
    }while(change<0);
    change=round(change); //get the round off value of change owed
    int coins=0;
    while(change>=25) //get number of quarters coins needed
    {
        change-=25;
        coins++;
    }
    while(change>=10) //get number of dimes coins needed
    {
        change-=10;
        coins++;
    }
    while(change>=5) //get number of nickels coins needed
    {
        change-=5;
        coins++;
    }
    while(change>=1) //get number of pennis coins needed
    {
        change-=1;
        coins++;
    }  
    printf("%i\n", coins); //output the number of coins needed ATQ
}

File: pset6/cash/cash.py
from cs50 import get_float

change=get_float("Change owed: ")

while change<0:
    change=get_float("Change owed: ")

change*=100
coins=0

while change>=25:
    change-=25
    coins+=1
while change>=10:
    change-=10
    coins+=1
while change>=5:
    change-=5
    coins+=1
while change>=1:
    change-=1
    coins+=1

print(coins)

